doc/conf.py

```diff
@@ -104,8 +103,8 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme_path = [os.path.join(os.path.dirname(catkin_sphinx.__file__), 'theme')]
-html_theme = 'ros-theme'
+# html_theme_path = [os.path.join(os.path.dirname(__file__), 'theme')]
+# html_theme = 'ros-theme'
```

Comment:
I left them commented out to be consistent with the rest of the file. This is a standard template, every other customizable value is commented out throughout the file to show examples of how to do it.